pallets/proposals/src/tests.rs

```diff
@@ -431,6 +431,12 @@ fn withdraw_removes_project_after_all_funds_taken() {
             RuntimeOrigin::signed(*ALICE),
             project_key
         ));
+
+        if let Some((account, projects)) = CompletedProjects::<Test>::iter().next() {
```

Comment:
withdraw_removes_project_after_all_funds_taken() should not test for completed projects. 
A new test would probably be appropriate here.

Reply:
And there should be a case that covers when you hit the bound MaxProjectPerAccount